### PR TITLE
(PUP-6172) Add static node config for huawei

### PR DESCRIPTION
**Pull request by LuvCurves**

### CODE REVIEW

**Comment by LuvCurves** [comment]
@johnduarte Close other PR, here is PR for stable branch

File: acceptance/config/nodes/huaweios-6-powerpc.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      - master
+    platform: el-7-x86_64
+    hypervisor: vcloud
+    template: redhat-7-x86_64
+  huaweios-6-powerpc:
+    roles:
+      - agent
+    platform: huaweios-6-powerpc
+    hypervisor: none
+    ip: huawei-ce6850-2-debian-vm-eth0.ops.puppetlabs.net
```
Comment (johnduarte) [inline acceptance/config/nodes/huaweios-6-powerpc.yaml]: The above is incorrect. The ip is set to the hostname. I believe it should look like the following:
```
ip: 10.0.7.26
vmhostname: huawei-ce6850-2-debian-vm-eth0.ops.puppetlabs.net
```

**Comment by johnduarte** [comment]
:+1:  LGTM @LuvCurves

**Comment by kylog** [comment]
LGTM.

Meta: @LuvCurves can you include "[skip ci]" in the commit message for PRs which only affect files under `acceptance`. This prevents appveyor and travis from kicking in, and since both those services can get backed up on us, it's a CI bandwidth win. Thanks!!

**Pull request merged**